src/ConnectedMode/Binding/NonRoslynBindingConfigProvider.cs

```diff
@@ -31,13 +31,28 @@
 
 namespace SonarLint.VisualStudio.ConnectedMode.Binding
 {
-    public class NonRoslynBindingConfigProvider : IBindingConfigProvider
+    internal class NonRoslynBindingConfigProvider : IBindingConfigProvider
     {
+        // List of languages that use this type of configuration file (all non-Roslyn languages)
+        internal static readonly IEnumerable<Language> SupportedLanguages = new []
```

Comment:
The list of languages supported by this class is an implementation detail of the class i.e. it shouldn't be set from outside. So, I added an `internal` constructor that allows the set of languages to be set when testing, and a `public` constructor that uses the default set of supported languages.

Reply:
I understand that it wasn't there before, but why is c# not in the list?